test/recipes/80-test_ssl_old.t

```diff
@@ -310,12 +329,14 @@ sub testss {
 }
 
 sub testssl {
-    my ($key, $cert, $CAtmp) = @_;
+    my ($key, $cert, $CAtmp, $provider, $configfile) = @_;
     my @CA = $CAtmp ? ("-CAfile", $CAtmp) : ("-CApath", bldtop_dir("certs"));
 
     my @ssltest = ("ssltest_old",
 		   "-s_key", $key, "-s_cert", $cert,
-		   "-c_key", $key, "-c_cert", $cert);
+		   "-c_key", $key, "-c_cert", $cert,
+           "-provider", $provider,
+           "-config", $configfile);
```

Comment:
The new lines are unaligned.  The previous lines are indented with tabs, so it's possible it looks right to you if you have 4 space tab translation, but...  and github doesn't exactly help...

Reply:
Fixed.